src/index.html

```diff
@@ -10,6 +10,7 @@
     <!-- Latest compiled and minified CSS -->
 
     <link href="favicon.ico" rel="icon" type="image/x-icon">
+    <link href="https://maxcdn.bootstrapcdn.com/bootstrap/4.0.0/css/bootstrap.min.css" rel="stylesheet">
```

Comment:
It looks like you added this line by yourself. This is not how Angular works. You add required CSS inside `angular.json`.